spiffworkflow-backend/src/spiffworkflow_backend/api.yml

```
@@ -3539,6 +3567,58 @@ components:
+    Pagination:
+      properties:
+        count:
+          type: integer
+          description: Number of items on current page
+        total:
+          type: integer
+          description: Total number of items
+        pages:
+          type: integer
+          description: Total number of pages
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)